config/samples/liberty.websphere.ibm.com_v1_webspherelibertyapplications.yaml

```diff
@@ -1,12 +1,37 @@
 apiVersion: liberty.websphere.ibm.com/v1
 kind: WebSphereLibertyApplication
 metadata:
-  name: websphereliberty-app-sample
+  name: wlo-app
 spec:
   license:
-    accept: false
-    edition: IBM WebSphere Application Server
-    productEntitlementSource: Standalone
-    metric: Virtual Processor Core (VPC)
+    accept: true
   applicationImage: registry.connect.redhat.com/ibm/open-liberty-samples:springPetClinic
-
+  expose: true
+  replicas: 3
+  probes:
+    liveness:
+      httpGet:
+        path: /
+        port: 9443
+        scheme: HTTPS
+      initialDelaySeconds: 70
+    readiness:
+      httpGet:
+        path: /
+        port: 9443
+        scheme: HTTPS
+      initialDelaySeconds: 60
+  resources:
+    limits:
+      cpu: '1'
+      memory: 1Gi
```

Comment:
Limit for ephemeral-storage is also required. Suggest using 500Mi for limit and 100Mi for request